### DISCUSSION TITLE
ESP32-S2 support #2

### DISCUSSION THREAD

**Discussion by AhsenDogan01** [schreibfaul1/ESP32-audioI2S#496] [Q&A]
Hi. This library works very well with ESP32-S. But when I use it with the ESP32-S2 there is some noise in the sound and the song plays a little slower than it should. How can I solve this problem? Does this library not support ESP32-S2?

**Comment by renatoianhez** [comment]
Hi! I couldn't get it to work on the Lolin S2 mini. Which pins did you use to connect the I2S?

**Comment by AhsenDogan01** [comment]
#define SD_CS         10
#define SPI_MOSI      11
#define SPI_MISO      13
#define SPI_SCK       12

#define I2S_DOUT      35
#define I2S_BCLK      36
#define I2S_LRC       37

In these definitions, I can play music with the "ESP32-S2-Soala" development board. But there is noise in music. Also, nothing changes when I define different i2s pins. Music plays again, but noise occurs. But when I do this with "ESP32S" there is no noise in the sound and it works just fine. And the codes are exactly the same.
